.coveragerc

```diff
@@ -3,3 +3,6 @@ exclude_lines =
     @abstract
     if TYPE_CHECKING:
     pragma: no cover
+[run]
+omit =
+  prawcore/_async/*.py
```

Comment:
This code needs tested.

Reply:
The code is the sole mirror of your synchronous part. And the synchronous part is tested.
WDYT?

Keep in mind that betamax is meant for Requests, and it has no support for async whatsoever.
If we create a mocking solution for the async part, it will be harder for us to maintain continuously. In fact it will be more complex than the project itself.

The most convenient way to verify that it is okay, is to run type check to ensure we did not miss an awaitable somewhere.

Reply:
The mirror could have bugs introduced so unfortunately not being tested won't work for this project. And yes, it is a lot of work to maintain a high level of test coverage.

Reply:
I usually work with the philosophy of if it's not tested, it doesn't work. Would VCR.py be able to record tests made by niquests? If so, then the cassettes could be pulled from Async PRAW for the async code tests.

Reply:
I understand, I will do my best. I need time to think of a solution.

> Would VCR.py be able to record tests made by niquests?

Yes, but it won't matter. The mocking tools out there don't support the awaitable part.

Reply:
I did what you asked.
The changes pushed just now add full coverage for the asynchronous part based on existing tests.

Everything should be in order.